aries-site/src/themes/aries.js

```diff
@@ -204,6 +212,18 @@ export const aries = deepMerge(hpe, {
       xxlarge: '166px',
     },
   },
+  tab: {
+    border: {
+      color: 'text-xweak',
```

Comment:
Default border is 2px. Design is showing a thin border bottom (0.5 px, but we will have to do 1px), and the active state has a 3px border. Not sure Grommet currently supports mixing the border sizes on Tab.

Reply:
I don't think it supports mixing border sizes. I couldn't find a way to do that last week. We should get clarity from the designers which px value they would like if it can only be one value for active and default. Good catch on the detail.